install_config/upgrading/manual_upgrades.adoc

```diff
@@ -911,53 +911,46 @@ Elasticsearch while running this and it will not perform an install.
 [[manual-upgrading-cluster-metrics]]
 == Upgrading Cluster Metrics
 
-Use the following to upgrade an
-link:../../install_config/cluster_metrics.html[already-deployed Cluster Metrics install].
+After upgrading an
+link:../../install_config/cluster_metrics.html[already-deployed Cluster Metrics install],
+you must update to a newer version of the metrics components.
+
+- The update process stops all the metrics containers,
+updates the metrics configuration files,
+and redeploys the newer components.
 
-To update to a newer version of the metrics components you will need to use the `refresh`
-mode of the deployer. This will stop all the metrics containers, update the metrics configuration
-files and redeploy the newer components. The `refresh` option will not change the metrics route or
-delete the metrics persistent volume claim.
+- It does not change the metrics route.
 
-Metrics stored to persistent volumes will still contain the persisted data when the updated
-metrics are brought back online. 
+- It does not delete the metrics persistent volume claim.
+Metrics stored to persistent volumes before the update
+are available after the update completes.
 
 [IMPORTANT]
 ====
-All non-persisted metric values will be lost during the metric `refresh`.
-
-All local changes to the metrics configurations will be overwritten with new configurations meant for the latest install.
-This includes things like number of instances in a replica set.
+The update deletes all non-persisted metric values,
+and overwrites local changes to the metrics configurations,
+e.g., number of instances in a replica set.
```

Comment:
Guidance here says to use e.g. sparingly, so maybe spell the abbreviation out: 
https://obriend.fedorapeople.org/WritingStyleGuide/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-E